lavalink/nodemanager.py

```diff
@@ -32,7 +34,21 @@ def add_node(self, host: str, port: int, password: str, region: str, name: str =
         """
         Adds a node
         ----------
-        TODO
+        :param host:
+            The host to which Lavalink server you're connecting to.
+        :param port:
+            The port to the Lavalink server you're connecting to.
+        :param password:
+            The password to the Lavalink server you're are connecting to, to authenticate with it.
+            Default password is ``youshallnotpass``.
+        :param region:
+            The region you want your node to connect to.
+        :param name:
+            The name of your node.
+        :param resume_key:
+            The resume key of the session you want to resume to.
+        :param resume_timeout:
+            The amount of time in seconds, that lavalink.py will reconnect to your node from a timeout.
```

Comment:
It sounds unclear to me, ll.py can still reconnect to a node even after this timeout. It's more like the time before you can't resume your session anymore

Reply:
You can copy the docstring from `client.py/add_node` if needed.